build.gradle.kts

```diff
@@ -84,8 +84,8 @@ jib {
 		labels = mapOf(
 				"maintainer" to "Krzysztof Koziol"
 		)
-		jvmFlags = listOf("-Xms512m", "-Djava.awt.headless=true")
+		jvmFlags = listOf("-Djava.awt.headless=true")
```

Comment:
`-Xms512m` is not needed?

Reply:
It's set in a different place like in the other services - two settings (-XX:InitialRAMPercentage=50, -XX:MaxRAMPercentage=75) should be a good replacement.